test/test_torch.py

```diff
@@ -7590,6 +7602,17 @@ def test_rot90(self, device):
         self.assertEqual(torch.tensor([4, 3, 2, 1]).view(2, 2), data.rot90(2, [0, 1]))
         self.assertEqual(torch.tensor([3, 1, 4, 2]).view(2, 2), data.rot90(3, [0, 1]))
 
+        def test_complex_rot90(dtype):
```

Comment:
same as above. we can just update this [test](https://github.com/pytorch/pytorch/blob/08304ccccc11e1a2152d52a8c74b5e88e71c2e6e/test/test_torch.py#L7586)

Reply:
Similar to above will rename this nested function to `_test_complex_rot90`